### PR TITLE
24 - Marketplace (1/2)

### LINKED ISSUE #24: Marketplace (1/2)
Top "Marketplace" section only (`Product` and `ProductImage` models)

https://docs.google.com/document/d/1Iiby1TCV4Ru035b8daerrvPXmQicXocspngtw7eTfN0/edit?usp=sharing

### PR DESCRIPTION
**Pull request by buckyroberts**
Closes #24

### CODE REVIEW

**Comment by buckyroberts** [comment]
@vosi Can you review this when you get a chance? It's for the first part of this feature https://docs.google.com/document/d/1Iiby1TCV4Ru035b8daerrvPXmQicXocspngtw7eTfN0/edit?usp=sharing

File: v1/products/serializers/product.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            'pk',
+            'description',
+            'price',
+            'published_at',
+            'status',
+            'title',
+            'product_images_meta'
+        )
+        model = Product
+        read_only_fields = 'created_date', 'modified_date'
```
Comment (vosi) [inline v1/products/serializers/product.py:44]: published_at should also be read_only

**Review by vosi — CHANGES_REQUESTED** [review]
1. no tests supplied
2. don't see the functionality for "published_at = DateTimeField()  # when the product was first set to status=active"
3. not sure that "We need an endpoint that will support bulk editing (updating multiple product images in 1 request) for ProductImage position" is implemented

File: v1/products/serializers/product.py (hunk 1 above)
Comment (manjyot) [reply v1/products/serializers/product.py:44]: @vosi Thanks for the feedback. I'll push the tests and these features shortly.

**Pull request closed without merging**